dandi/organize.py

```diff
@@ -789,13 +783,15 @@ def act(func, *args, **kwargs):
             )
         lgr.info("We will organize %s in-place", dandiset_path)
         in_place = True
-        paths = dandiset_path
+        paths = [dandiset_path]
 
     if len(paths) == 1 and paths[0].endswith(".json"):
         # Our dumps of metadata
         metadata = load_jsonl(paths[0])
+        link_test_file = metadata[0]["path"]
```

Comment:
in principle AFAIK we do not even need `link_test_file` since we can simply use `metadata[0]["path"]` directly since we do load metadata records for regular paths too... but ok, let it be this way for now